angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-details.html

```diff
@@ -10,111 +11,90 @@
+                <ul>
+                    <li ng-repeat="keyword in portlet.keywords | orderBy:'toLowerCase()'  track by $index"><a href='apps/search/{{ ::keyword }}'>{{ ::keyword }}</a></li>
+                </ul>
+            </div>
+        </div>
 	</div>
-	<div class="row no-margin details-header"  ng-if='!loading && !error'>
-		<div class="col-xs-12 col-sm-4 col-lg-3 desc-item">
-			<h3 tabindex="0" class="center">Description</h3>
-			<p tabindex="0" class="app-description">{{::portlet.description}}</p>
-		</div>
-		<div class="col-xs-12 col-sm-4 col-lg-3 desc-item center rating-div">
-			<h3 tabindex="0">Ratings</h3>
-			<span><rating ng-model="portlet.rating" readonly="true" class="rating"></rating>
-    ( {{::portlet.userRated}} rating<span ng-if="portlet.userRated !== 1">s</span> )
-  		</span><br>
-  		<button ng-hide="GuestMode" ng-click="openRating('sm',portlet.fname, portlet.title)" class="btn btn-default rate-button">Rate</button>
-		</div>
-		<div class="col-xs-12 col-sm-4 col-lg-3 desc-item center">
-			<h3 tabindex="0">Related Apps</h3>
-      <ul>
-        <li ng-repeat="related in portlet.relatedPortlets"><a href="apps/details/{{ ::related.fname }}">{{ ::related.title }}</a></li>
-      </ul>
-		</div>
-		<div class="col-xs-12 col-sm-4 col-lg-3 desc-item center">
-			<h3 tabindex="0">Categories</h3>
-			<a ng-repeat="category in portlet.categories" href="apps" ng-click="specifyCategory(category)" class="btn btn-outline btn-sm category-links">{{::category}}</a>
-		</div>
-		<div class="col-xs-12 col-sm-4 col-lg-3 desc-item center" ng-if="$storage.showKeywordsInMarketplace">
-			<h3>Keywords</h3>
-	    <ul>
-	      <li ng-repeat="keyword in portlet.keywords | orderBy:'toLowerCase()'  track by $index"><a href='apps/search/{{ ::keyword }}'>{{ ::keyword }}</a></li>
-	    </ul>
-		</div>
-		<div class="col-xs-12 col-lg-9 desc-item screenshots portlet-screenshots" ng-if="portlet.marketplaceScreenshots.length != 0">
-			<h3>Screenshots</h3>
-	    <ul class="enlarge">
-	      <li ng-repeat="screenshot in portlet.marketplaceScreenshots">
-	        <img ng-src="{{ ::screenshot.url }}" width ="250px" alt="{{ ::screenshot.captions }}">
-	        <span>
-	        	<img ng-src="{{ ::screenshot.url }}" alt="{{ ::screenshot.captions }}">
-	        	<p ng-repeat="caption in screenshot.captions">{{ ::caption }}</p>
-	        </span>
-	      </li>
-	    </ul>
-		</div>
-	</div>
-
-  <div class='preview'>
-    <h2>Widget Preview</h2>
-    <div class='middle'>
-      <widget fname="portlet.fname"></widget>
-    </div>
-  </div>
 
-	<div class="portlet-footer left">
-	 	<ul>
-	 		<li><a ng-href="{{backURL}}"><i class="fa fa-arrow-circle-o-left"></i> Back to {{backText}}</a></li>
-	 	</ul>
+	<div class="portlet-footer" layout="row" layout-align="start center">
+        <md-card-actions>
+            <md-button ng-href="{{backURL}}"><i class="fa fa-arrow-circle-o-left"></i> Back to {{backText}}</md-button>
```

Comment:
Maybe an `aria-hidden="true"` on the `<i .. fa-arrow-circle-o-left...`? It's maybe not strictly speaking "decoration", but I don't think it adds enough meaning beyond the "Back to" text, so screen readers would still be best served hiding the icon?